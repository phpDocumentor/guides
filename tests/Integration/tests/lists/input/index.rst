
*    List item 1
*    This is list item two
*    And finally, list item three.

*    Another list
    *    that is indented
    *    indented list item