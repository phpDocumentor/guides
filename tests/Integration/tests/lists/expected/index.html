<ul class="simple">
    <li><p>List item 1</p></li>
    <li><p>This is list item two</p></li>
    <li><p>And finally, list item three.</p></li>
</ul>
<ul class="simple">
    <li><p>Another list</p>
        <ul>
            <li><p>that is indented</p></li>
            <li><p>indented list item</p></li>
        </ul>
    </li>
</ul>